### ISSUE TITLE
How to use Apex to make subnetwork using FP16 and another subnetwork using FP32??

### ISSUE THREAD

**Issue by youngfly11** [NVIDIA/apex#1234]

**Comment by ptrblck** [comment]
`apex.amp` is deprecated and you should use the native implementation via `torch.cuda.amp` as described [here](https://github.com/NVIDIA/apex/issues/818)

**Comment by youngfly11** [comment]
> `apex.amp` is deprecated and you should use the native implementation via `torch.cuda.amp` as described [here](https://github.com/NVIDIA/apex/issues/818)

Hi, ptrblck;

Thanks for your response. Using native torch.cuda.amp is a good choice. But there is another question that:

Does the current native amp in PyTorch support gradient checkpointing and mixed precision training at the same time?
